License2Deploy/rolling_deploy.py

```diff
@@ -222,11 +223,47 @@ def healthcheck_new_instances(self, group_name): # pragma: no cover
     self.wait_for_new_instances(new_instance_ids) #Wait for new instances to be up and ready
     self.lb_healthcheck(new_instance_ids) #Once instances are ready, healthcheck. If successful, decrease desired count.
 
+  def retrieve_all_cloudwatch_alarms(self):
+    """ Retrieve all the Cloud-Watch alarms for the given project and environment """
+    try:
+      cloud_watch_alarms = self.conn_cloudwatch.describe_alarms()
+    except Exception as e:
+      logging.error("Error while retrieving the list of cloud-watch alarms. Error: {0}".format(e))
+      exit(self.exit_error_code)
+    cloud_watch_alarms = filter(lambda alarm: self.project in alarm.name and self.env in alarm.name, cloud_watch_alarms)
+    if len(self.cloud_watch_alarms) == 0:
+       logging.info("No cloud-watch alarm found")
+    return cloud_watch_alarms
+
+  def disable_all_cloudwatch_alarms(self):
```

Comment:
This method name (and others for retrieve and enable) is a little misleading. Are you actually disabling ALL cloudwatch alarms?
I hope not ;)